<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HeartPlace</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditName">
         <property name="styleSheet">
          <string notr="true">border-radius: 2px;</string>
         </property>
         <property name="placeholderText">
          <string> 姓名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGender">
         <property name="styleSheet">
          <string notr="true">border-radius: 2px;</string>
         </property>
         <property name="placeholderText">
          <string> 性别：M/F</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPhone">
         <property name="styleSheet">
          <string notr="true">border-radius: 2px;</string>
         </property>
         <property name="placeholderText">
          <string> 手机号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditBirth">
         <property name="styleSheet">
          <string notr="true">border-radius: 2px;</string>
         </property>
         <property name="placeholderText">
          <string> 生日: 1900-01-01</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditLogDate">
         <property name="styleSheet">
          <string notr="true">border-radius: 2px;</string>
         </property>
         <property name="placeholderText">
          <string> 签到日期: 1900-01-01</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>保存</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionreport"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);</string>
   </property>
  </widget>
  <action name="actionreport">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons8-Data Grid-96.png</normaloff>:/icons8-Data Grid-96.png</iconset>
   </property>
   <property name="text">
    <string>report</string>
   </property>
   <property name="iconText">
    <string>报表</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
